ibc-testkit/src/fixtures/clients/tendermint.rs

```diff
@@ -117,7 +117,7 @@ pub fn dummy_tendermint_header() -> tendermint::block::Header {
 
     serde_json::from_str::<SignedHeader>(include_str!(concat!(
         env!("CARGO_MANIFEST_DIR"),
-        "/tests/data/json/signed_header.json"
+        "/../tests-integration/tests/data/json/signed_header.json"
```

Comment:
Interesting! If `ibc-testkit` is offering these methods, then these json files should live inside `ibc-testkit`.

Note: the above json files for `AbciQuery` serde tests should live in `tests-integration` - as they are not part of our public API.

Reply:
[1065055](https://github.com/cosmos/ibc-rs/pull/1233/commits/1065055635c4e79ef3fcb17cbd358fc62b4a1165)